llms/mlx_lm/utils.py

```diff
@@ -235,6 +267,10 @@ def _step(y):
             for processor in logits_processor:
                 logits = processor(tokens, logits)
 
+        prompt_cache = maybe_quantize_kv_cache(
+            prompt_cache, quantized_kv_start, kv_group_size, kv_bits
+        )
```

Comment:
Going from the comment above this could simply become:

```python
maybe_quantize_kv_cache(
    prompt_cache, quantized_kv_start, kv_group_size, kv_bits
)
```